src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -47,6 +52,28 @@ public async Task<IActionResult> GetQuestionBySlug(string sectionSlug,
         return await router.ValidateRoute(sectionSlug, questionSlug, this, cancellationToken);
     }
 
+    [LogInvalidModelState]
+    [HttpGet("question/preview/{questionId}")]
+    public async Task<IActionResult> GetQuestionPreviewById(string questionId,
+                                                            [FromServices] ContentfulOptions contentfulOptions,
+                                                            CancellationToken cancellationToken = default)
+    {
+        if (!contentfulOptions.UsePreview)
+            return new RedirectResult("/self-assessment");
+
+        var question = await _getEntityFromContentfulQuery.GetEntityById<Question>(questionId, cancellationToken) ??
```

Comment:
Other thought:

If we _need_ the section then you can _probably_ just do a Contentful query to
1. Retrieve the section, where it has a matching question
2. Filter out what we return from the API, so that we only return the necessary info we need from the section + question

Reply:
Resolving since we don't need it, good shout